packages/react-scripts/config/webpack.config.js

```diff
@@ -117,6 +117,7 @@ module.exports = function(webpackEnv) {
                 flexbox: 'no-2009',
               },
               stage: 3,
+              browsers: appPackageJson.browserslist,
```

Comment:
Hi @mingenesis 👋 

Could you help me to replicate the bug that this PR fixes? On that note, `postcss-preset-env` states the following in its documentation:

> PostCSS Preset Env supports any standard browserslist configuration, which can be a .browserslistrc file, a browserslist key in package.json, or browserslist environment variables.
> The browsers option should only be used when a standard browserslist configuration is not available.
https://github.com/csstools/postcss-preset-env#browsers

Is that not working as intended?

Have a good day :)

Reply:
Hi @tobiasbueschel 😄
The css file from outside(maybe in "node_modules/some-ui") will parse the browserslist from its own project("node_modules/some-ui/.browserslistrc" or "node_modules/some-ui/package.json"). It will not parse the browserslist in the cra project. This is not my expectation.